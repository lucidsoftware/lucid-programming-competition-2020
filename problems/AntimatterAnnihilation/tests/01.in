more_antiantimattermatter_to_annihilate