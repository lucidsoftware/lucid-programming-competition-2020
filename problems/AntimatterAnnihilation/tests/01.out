more__to_annihilate